Providers/Scripts/3.x/Scripts/nxAutomationWorker.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+# ====================================
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# See license.txt for license information.
+# ====================================
+import os
+import sys
+
+
+def init_locals(Name):
+    if Name is None:
+        Name = ''
+    return Name.encode('ascii', 'ignore')
```

Comment:
This will convert the string to bytes in python 3.x
The 'encode' is not wanted here for parameters.